<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createTable</class>
 <widget class="QWidget" name="createTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="closeBut">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/createDB/png/resources/closePNG.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>45</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tableName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>45</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>550</y>
     <width>170</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/createtable/jpg/resources/createTable.jpg)}
QPushButton:pressed{border-image: url(:/createtable/jpg/resources/createTable_pressed.jpg)} </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>880</width>
     <height>400</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="flashButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>45</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/createtable/png/resources/reflash.png)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>281</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
        border: none;
        color: white;
        text-align: center;
        background: rgb(68, 69, 73);
}
QProgressBar::chunk {
        border: none;
        background: rgb(0, 160, 230);
}</string>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
